app/views/application_letters/check.html.erb

```diff
@@ -0,0 +1,64 @@
+<%- model_class = ApplicationLetter -%>
+
+<div class="page-header">
+  <h1><%=t '.check_application_for', event_name: @application_letter.event.name %></h1>
+  <p class='alert alert-info' >
+    <% if @application_deadline_exceeded %>
+      <%= t('.deadline_exceeded') %>
+    <% else %>
+      <%= t('.can_change_until', application_deadline: l(@application_letter.event.application_deadline)) %>
+    <% end %>
+  </p>
+</div>
+
+<h3><%= t('.my_application') %></h3>
+<div id='application_details'>
+  <strong><%= ApplicationLetter.human_attribute_name(:grade) %>:</strong>
+  <%= @application_letter.grade %><br/>
+  <strong><%= ApplicationLetter.human_attribute_name(:motivation) %>:</strong><br/>
+  <%= @application_letter.motivation %><br/>
+  <strong><%= ApplicationLetter.human_attribute_name(:experience) %>:</strong><br/>
+  <%= @application_letter.experience %><br/>
+  <strong><%= ApplicationLetter.human_attribute_name(:coding_skills) %>:</strong><br/>
+  <%= @application_letter.coding_skills %><br/>
+  <strong><%= ApplicationLetter.human_attribute_name(:emergency_number) %>:</strong>
+  <%= @application_letter.emergency_number %><br/>
+
+  <% habits = Array.new
+    if @application_letter.vegeterian
+      habits.push(ApplicationLetter.human_attribute_name(:vegeterian))
+    end
+    if @application_letter.vegan
+      habits.push(ApplicationLetter.human_attribute_name(:vegan))
+    end
+    if @application_letter.allergic
+      habits.push(ApplicationLetter.human_attribute_name(:allergic))
+    end
+  %>
```

Comment:
Maybe a model method for the application_letter that returns the habits as a list?